### PR TITLE
[MIG] base_technical_features: Migration to 16.0

### PR DESCRIPTION
**Pull request by apineux**
#534

### CODE REVIEW

**Review by baimont — APPROVED** [review]

File: base_technical_features/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Technical features group",
     "summary": "Access to technical features without activating debug mode",
-    "version": "15.0.1.1.0",
+    "version": "16.0.1.0.0",
```
Comment (sebalix) [inline base_technical_features/__manifest__.py:6]: Better to put the version number update in the migration commit

**Review by AungKoKoLin1997 — COMMENTED** [review]
Function review: It worked as expected.

**Review by legalsylvain — APPROVED** [review]

**Review by JoanSForgeFlow — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Pull request merged**